majilove/models.py

```diff
@@ -100,9 +100,9 @@ def display_blood_type(self):
     HOBBYS_CHOICES = ALL_ALT_LANGUAGES
     d_hobbys = models.TextField(_('Hobby'), null=True)
 
-    image = models.ImageField(_('Image'), upload_to=uploadItem('idol'))
+    image = models.ImageField(_('Image'), upload_to=uploadItem('idol'), null=True)
```

Comment:
Why is it not required anymore?

Reply:
Mostly to make my life easier while doing testing